src/styles/review.scss

```diff
@@ -0,0 +1,32 @@
+.review-block {
+  display: flex;
+  margin-top: 15px;
+
+  &__stars {
+    display: flex;
+    width: 91px;
+    height: 15px;

```

Comment:
not fixed

Reply:
Don't understand what to fix here? make variables?

Reply:
you don't need height here